File: app/src/main/java/com/dicoding/moviecataloguerv/data/source/remote/response/ImageResponse.java
package com.dicoding.moviecataloguerv.data.source.remote.response;

import com.dicoding.moviecataloguerv.data.source.model.ImageItems;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import lombok.Data;

@Data
public class ImageResponse {

    @SerializedName("backdrops")
    @Expose
    private List<ImageItems> backdrops;
}

File: core/src/main/kotlin/com/ryanrvldo/core/data/source/remote/response/commons/ImageResponse.kt
package com.ryanrvldo.core.data.source.remote.response.commons

import com.google.gson.annotations.SerializedName

data class ImageResponse(
    @SerializedName("aspect_ratio") val aspectRatio: Double?,
    @SerializedName("file_path") val filePath: String?,
    @SerializedName("vote_average") val rating: Double?,
    @SerializedName("vote_count") val ratingCount: Int?,
    @SerializedName("height") val height: Int?,
    @SerializedName("width") val width: Int?,
)
